language/analysis/src/main/java/com/google/cloud/language/samples/AnalyzeBeta.java

```diff
@@ -16,44 +16,30 @@
 
 package com.google.cloud.language.samples;
 
-
 import com.google.cloud.language.v1beta2.AnalyzeEntitySentimentRequest;
 import com.google.cloud.language.v1beta2.AnalyzeEntitySentimentResponse;
```

Comment:
AnalyzeEntitySentiment has been moved to v1; the samples need to be moved to Analyze.java.

Reply:
Will be moved when library is regenerated.

Reply:
library published to maven. (0.23.1-beta)